TACHO_DRIVER.md

```diff
@@ -0,0 +1,233 @@
+Tacho Driver Update Proposal
```

Comment:
Do you intend on leaving this document when you squash? If so, it needs to be cleaned up to reflect what was actually done.